style.css

```diff
@@ -0,0 +1,7 @@
+body {
```

Comment:
- [ ] I suggest you to add some code for "reset" the default padding and borders in the browsers, and use, for example, the box-sizing:border-box property in order to avoid issues in your box-model items (Optional).